src/main/java/seedu/address/storage/JsonAdaptedEatery.java

```diff
@@ -32,9 +34,11 @@
     @JsonCreator
     public JsonAdaptedEatery(@JsonProperty("name") String name,
                              @JsonProperty("address") String address,
+                             @JsonProperty("category") String category,
```

Comment:
good that you remembered to update how the Json file is being read!

Reply:
what the fuck